.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: cpp
+sudo: false
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: BUILD_SYSTEM=bazel
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - wget
+            - pkg-config
```

Comment:
Looks like a legacy from times when bazel was to be build before use. Will remove.